src/pools/reserve.ts

```diff
@@ -204,7 +204,7 @@ export class JetReserve {
       pythOracle.priceData,
       pythOracle.productData,
       mintInfo.data,
-      new BN(availableLiquidity)
+      new TokenAmount(new BN(availableLiquidity), parseMintAccount(mintInfo.data).decimals, data.tokenMint)
```

Comment:
You are parsing the mint twice, once in this method and once in decodeReserveData (Same decoding the mint twice in loadMultiple()). I recommend you do each step once..
```suggestion
      new BN(availableLiquidity)
```